code/modules/power/apc/apc_tool_act.dm

```diff
@@ -1,6 +1,21 @@
 //attack with an item - open/close cover, insert cell, or (un)lock interface
 /obj/machinery/power/apc/crowbar_act(mob/user, obj/item/crowbar)
 	. = TRUE
+
+		//Prying off broken cover
+	if(opened == APC_COVER_CLOSED || opened == APC_COVER_OPENED && (machine_stat & BROKEN))
+		crowbar.play_tool_sound(src)
+		user.visible_message(span_notice("[user.name] begins prying off the broken [name] cover!"))
+		balloon_alert(user, "you begin prying")
```

Comment:
ditto on both, also this is a better balloon alert

```suggestion
		balloon_alert(user, "prying...")
```

Reply:
if you really want a message to show up to everyone do `balloon_alert_to_viewers` but that's at your discretion